animal-trading-cards/normalize.css

```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
mention it's from third party

Reply:
done

https://github.com/sebz996/front-end-web-developer-course/pull/1/commits/2b97e312bd4557eb3223ac1692571df2544caf80